meson.build

```diff
@@ -2344,13 +2344,13 @@ if git.found()
                  ':/*.[ch]'])
         all_files = files(all_files.stdout().split())
 
-        run_target(
+        custom_target(
                 'tags',
-                input : all_files,
+                output : 'tags',
                 command : ['env', 'etags', '-o', '@0@/TAGS'.format(meson.source_root())] + all_files)
-        run_target(
+        custom_target(
```

Comment:
I'm not sure about `tags`, but `ctags` seems to be broken due to https://github.com/mesonbuild/meson/issues/3589. Maybe `--tag-relative=yes` could help to get around the issue.

Reply:
`tags` seems to work fine here. I'm not even sure how to test `ctags` (it's used by `vi`, right?). Please open a new issue with more details, otherwise this will get lost.